common/domain/replication_queue.go

```diff
@@ -336,13 +336,14 @@ func (q *replicationQueueImpl) emitDLQSize() {
 				).IncCounter(
 					metrics.DomainReplicationQueueSizeErrorCount,
 				)
+			} else {
+				q.metricsClient.Scope(
+					metrics.DomainReplicationQueueScope,
+				).UpdateGauge(
+					metrics.DomainReplicationQueueSizeGauge,
+					float64(size),
+				)
 			}
-			q.metricsClient.Scope(
```

Comment:
nit: lets make sure it wont make the existing alerts going off.

Reply:
I updated the alert to use this one.